pages/price-feeds/use-real-time-data/evm.mdx

```diff
@@ -70,21 +48,38 @@ import "@pythnetwork/pyth-sdk-solidity/PythStructs.sol";
 contract SomeContract {
   IPyth pyth;
 
+  /**
+   * THIS IS AN EXAMPLE CONTRACT. IT IS NOT INTENDED TO BE USED IN PRODUCTION.
+   * THIS CONTRACT IS FOR DEMONSTRATION PURPOSES ONLY.
+   */
+
+  /**
+      * Instantiate IPyth interface with the address of the Pyth contract.
+      * Pyth contract address can be found at https://docs.pyth.network/price-feeds/contract-addresses/evm.
+
+      * @param pythContract The address of the Pyth contract
+      */
   constructor(address pythContract) {
     pyth = IPyth(pythContract);
   }
 
-  function doSomething(
-    uint someArg,
-    string memory otherArg,
-    bytes[] calldata priceUpdateData
-  ) public payable {
-    // Update the on-chain Pyth price(s)
+  /**
+     * This function is an example of how to interact with the Pyth contract.
+
+     * Fetch the updateData from an off-chain service and pass it to the Pyth contract to update the price feeds.
```

Comment:
the phrase "an off-chain service" just begs the question "what off-chain service?"

this is going to be confusing